pkgs/by-name/sa/satellite/package.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, python3
+, fetchFromGitea
+, gobject-introspection
+, gtk3
+, libhandy
+, modemmanager
+, wrapGAppsHook
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "satellite";
+  version = "0.4.2";
+
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
we should use the new builder. It also works with older projects still using setup.py.

---

Thanks for the great suggestion!